Sources/SBGCore/EnvironmentInitialization/DemoFilesContentProvider.swift

```diff
@@ -0,0 +1,102 @@
+//
+// Created by Karolina Samorek on 2018-10-31.
+//
+
+import Foundation
+
+final class DemoFilesContentProvider: FilesContentProvider {
+
+    private(set) var sbgConfigFileContent: String = """
+                                                    {
```

Comment:
Do you need such big indentation?

Reply:
imo it looks better when indentation is aligned with `"""` ;)